### PR TITLE
Shrink it, use arrows for instructions

### PR DESCRIPTION
**Pull request by tnunamak**
@sirugh @Atlas- can we have someone double check PRs before merging them? If you like it, `:+1:` and merge it :smile_cat:!

### CODE REVIEW

**Comment by chrispound** [comment]
Really like the instructions becoming arrows.

what are you're thoughts on the  white space to the right of the execute button in the evil plan box?

**Comment by tnunamak** [comment]
I'm not sure.. the UI definitely isn't done. It does fill out when you start to make a selection. I think it's better to put the execute button close to the selection instead of moving the whitespace between them.

File: robits.js [hunk 1]
```diff
@@ -92,11 +91,13 @@ function displayPossibleMoves() {
...
+
+    function getArrow(id, direction) {
+      return $("<img id='"+ id +"' data-move='" + direction + "' data-src='assets/arrow-" + direction + ".png' src='assets/arrow-" + direction + ".png' class='img-rounded amove' alt='" + direction + "'>");
+    }
     _.each(generateNewMoves(), function(move) {
       imgId++;
-      possibleMovesDiv.append(
-            "<img id='move"+ imgId +"' data-move='" + move + "' data-src='assets/arrow-" + move + ".png' src='assets/arrow-" + move + ".png' class='img-rounded amove' alt='" + move + "' style='width: 96px; height: 96px;'>"
-        );
+      possibleMovesDiv.append(getArrow('move'+imgId, move));
```
Comment (sirugh) [inline robits.js:109]: The naming here suggests a single direction and doesn't account for rotations or additional distances.

Comment (tnunamak) [inline robits.js:109]: The naming of what? getArrow?

Comment (sirugh) [inline robits.js:109]: Yea.

**Comment by tnunamak** [comment]
Better?

**Pull request merged**